apache-ansible-demo/docs/content.md

```diff
@@ -0,0 +1,13 @@
+The resource package contains resource definitions and scripts that allow to manage an Apache server on OpenStack through the Ansible Lifecycle Driver.
+
+The package contains the descriptor of an assembly ([apache-ansible-demo](./Descriptor/assembly.yml)) and of a resource ([hw-apache](./Contains/hw-apache-vnfc/Definitions/lm/resource.yaml)).  
+The resource descriptor defines properties of the Apache server and of the underlying infrastructure. It also defines the characteristics of the lifecycle execution that is used to manage the resource.  
+The lifecycle definition indicates that Adopt, Create, Configure, Install, Start, Stop and Delete intents are defined for the resource and that are managed by the Ansible Lifecycle Driver.  
```

Comment:
Is it more accurate to refer to these as Adopt, Create, Configure, Install, Start, Stop and Delete **transitions** ,rather than intents?